pkgs/tools/security/enpass/default.nix

```diff
@@ -0,0 +1,103 @@
+{stdenv, fetchurl, dpkg, patchelf, openssl, xorg
+, glib, mesa, libpulseaudio, zlib, dbus, fontconfig, freetype
+, gtk2, pango, atk, cairo, gdk_pixbuf, jasper, xkeyboardconfig
+, makeWrapper , makeDesktopItem, python, pythonPackages}:
+let
```

Comment:
If you want to support restrict platforms, you can add this line:

`assert stdenv.system == "i686-linux" || stdenv.system == "x86_64-linux";`